assets/src/components/p4_button/edit.js

```diff
@@ -43,7 +43,7 @@ import { link } from '@wordpress/icons';
 const { getComputedStyle } = window;
 
 const p4_button_text_colors = [
-  { name: 'dark-shade-black', color: '#1a1a1a' },
+  { name: 'grey-80', color: '#1a1a1a' },
```

Comment:
I think I'm missing something, shouldn't you change `#1a1a1a` to `#020202` here, as you wrote in the styleguide ?

Reply:
NIce catch!